database/src/main/postgres/public/V1.2.2__jsonb_array_to_text_array.sql

```diff
@@ -30,10 +29,10 @@ CREATE OR REPLACE FUNCTION public.jsonb_array_to_text_array(
 --      Postgres array
 --
 -------------------------------------------------------------------------------
-    LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE
-BEGIN ATOMIC
-    SELECT ARRAY(SELECT jsonb_array_elements_text(i_json_array));
-END;
-
+LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE
+AS
+    $$
+        SELECT ARRAY(SELECT jsonb_array_elements_text(i_json_array));
+    $$;
 
-GRANT EXECUTE ON FUNCTION public.jsonb_array_to_text_array(JSONB) TO public;
+GRANT EXECUTE ON FUNCTION public.jsonb_array_to_text_array(JSONB) TO public;
```

Comment:
no newline at the end of file

Reply:
Ok, will add it, once done I will update this comment.

Reply:
done